java/src/org/openqa/selenium/remote/RemoteWebDriver.java

```diff
@@ -562,8 +562,8 @@ protected Response execute(CommandPayload payload) {
         } else {
           toThrow =
               new SessionNotCreatedException(
-                  "Possible causes are invalid address of the remote server or browser start-up"
-                      + " failure.",
+                  "Possible causes are invalid address of the remote server, insufficient"
+                      + " capabilities provided or browser start-up failure.",
```

Comment:
What type of insufficient capabilities would cause this?

Reply:
I believe any kind of insufficient capabilities will cause this as it seems to be in https://github.com/SeleniumHQ/selenium/issues/14125

Reply:
Thank you for sharing. I think the error emphasises sharing the capabilities causing the error, which might not be possible or need to be shared as part of the exception. This message does not capture enough details to fix that issue and it also adds to the existing ambiguity. I am going to close this PR. Please look into how the capabilities causing the error can be shared out. Thank you!